packages/ChangesReloaded-Tests.package/CRLinter.class/instance/testAddDependantMethodsLessOrEqualThanRemoveDependents.st

```diff
@@ -0,0 +1,4 @@
+test-for-possible-bugs
```

Comment:
What's the different between `testForBugs` and `test-for-possible-bugs`?

Reply:
SwaLint categorizes its lint rules into "Bugs" and "Possible bugs"

Reply:
Nice job SWA-Lint